src/main/java/Main.java

```diff
@@ -0,0 +1,59 @@
+import core.basesyntax.model.FruitTransaction;
```

Comment:
The source file is missing a package statement. Please add the appropriate package statement at the beginning of the file. For example, you can add `package core.basesyntax;` at the beginning of the file to include the file in the `core.basesyntax` package. 📦